python/pfs/utils/fiberids.py

```diff
@@ -87,14 +89,38 @@ def load(self, path):
                  ('fiberHoleId', 'u2'),
                  ('scienceFiberId', 'u2'),
                  ('fiberId', 'u2'),
-                 ('sunssFiberId', 'u2'),
-                 ('USCONNECMTPAId', 'U15'),
-                 ('USCONNECMTPCId', 'U15'),
-                 ('USCONNECMTPBAId', 'U15'),
-                 ('USCONNECMTPBCId', 'U15'),
+                 ('sunssFiberId', 'U4'),
+                 ('mtp_A', 'U15'),
+                 ('mtp_PC', 'U15'),
+                 ('mtp_BA', 'U15'),
+                 ('mtp_BC', 'U15'),
                  ]
 
         self.data = np.genfromtxt(flist[0], dtype=dtype,
+                                  missing_values='-',
```

Comment:
I think I'd prefer to return arrays with {{None}} set for invalid values rather than using MaskedArrays;  for example {{set(values)}} doesn't work for masked arrays.  If you try to use `array.filled` you can't set the bad fields to None, which is the natural value at least for non-floating post fields.

Reply:
Although sunssid returns a masked array all the fields are good, even those marked `emp`.  If we are going to use masked arrays let's use them everywhere.

Reply:
OK, I need to think about this. There does not seem to be any support for passing ``None`` as a filling value, but will check some more later. In any case, I agree that ``emp`` values for the sunssid should have been treated as a missing value in addition to ``-``. This was a mistake and corrected.

Reply:
I think if you don't use `missing_values` and avoid masked images it's simple enough to use None.  It's what we do almost everywhere else in the PFS code base, and I think we should do so here too.

Reply:
``MaskedArray`` s are reverted. The issue regarding ``None`` is related to integer fields. We've accepted that 65535 is file as the fill value.